argon2_test.go

```diff
@@ -41,25 +40,25 @@ type argon2Suite struct {
 
 	kdf testutil.MockArgon2KDF
 
-	halfTotalRamKiB uint32
-	cpus            uint8
+	maxMemoryCostKiBAuto int
+	cpusAuto             int
 }
 
 func (s *argon2Suite) SetUpSuite(c *C) {
 	var sysInfo unix.Sysinfo_t
 	c.Check(unix.Sysinfo(&sysInfo), IsNil)
 
 	halfTotalRamKiB := uint64(sysInfo.Totalram) * uint64(sysInfo.Unit) / 2048
-	if halfTotalRamKiB > math.MaxUint32 {
-		halfTotalRamKiB = math.MaxUint32
+	if halfTotalRamKiB > math.MaxInt32 {
+		halfTotalRamKiB = math.MaxInt32
 	}
-	s.halfTotalRamKiB = uint32(halfTotalRamKiB)
+	s.maxMemoryCostKiBAuto = int(halfTotalRamKiB)
```

Comment:
I'm probably confused, we set this but we don't seem to read it anywhere ?

Reply:
You're right, this value appears to be unused.